collectors/osv/collectors.py

```diff
@@ -166,12 +168,15 @@ def save_snippet(
             return 0, 0
 
         if not cve_ids:
-            _, created = Snippet.objects.update_or_create(
+            # This keeps the structure consistent and ease snippets filtering without cve_id
+            content["cve_id"] = None
+            snippet, created = Snippet.objects.update_or_create(
                 source=Snippet.Source.OSV,
                 external_id=osv_id,
                 defaults={"content": content},
             )
             if created:
+                snippet.convert_snippet_to_flaw()
```

Comment:
There is probably an issue when a flaw without `cve_id` is already created in BZ, but we lose the OSIDB data. In this case, even though flaws are downloaded before the OSV collector runs, we do not have a way to say that this newly created snippet should be connected to an already existing flaw in BZ (as there is no `cve_id`). Thus, we will create a new (duplicate) flaw in BZ. We can discuss this tomorrow.